contentcuration/contentcuration/not_production_settings.py

```diff
@@ -10,7 +10,10 @@
 ALLOWED_HOSTS = ["studio.local", "192.168.31.9", "127.0.0.1", "*"]
 
 ACCOUNT_ACTIVATION_DAYS = 7
-EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = 'postmark.django_backend.EmailBackend'
```

Comment:
Setting to use this in debug mode so we can catch potential future issues with sending emails